CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - [#168](https://github.com/smile-io/ember-polaris/pull/168) [INTERNAL] Upgrade to Ember CLI `3.4.2`
 - [#171](https://github.com/smile-io/ember-polaris/pull/171) [UPDATE] Shopify Polaris `v2.11.0`
 - [#174](https://github.com/smile-io/ember-polaris/pull/174) [FEATURE] Add `polaris-inline-error` component
+- [#191](https://github.com/smile-io/ember-polaris/pull/191) [ENHANCEMENT] Layout's annotated section to use a new css selector instead of subdued text style for description
```

Comment:
Nit: this description is pretty complex for those just wanting to have an overview of what's happened. How about something like `Update `polaris-layout/annotated-section` styling to match Polaris v2.11.0` instead?